app/src/main/java/com/sdp/movemeet/view/profile/EditProfileActivity.java

```diff
@@ -108,8 +110,7 @@ protected void onActivityResult(int requestCode, int resultCode, @androidx.annot
             if (resultCode == Activity.RESULT_OK) {
                 Uri imageUri = data.getData();
                 progressBar.setVisibility(View.VISIBLE);
-                String imagePath = "users/" + userId + "/profile.jpg";
-                FirebaseInteraction.uploadImageToFirebase(storageReference, imagePath, imageUri, profileImage, progressBar);
+                FirebaseInteraction.uploadImageToFirebase(storageReference, userImagePath, imageUri, profileImage, progressBar);
```

Comment:
Please try to not use `FirebaseInteraction` so we can delete it :(

Reply:
Yes this is indeed what we are fighting for at the moment! Sorry for that, but Mark is normally responsible for implementing the backend abstraction in EditProfileActivity. In case it is not properly done, I will have to personally update this EditProfileActivity in the next sprint!